Issue #2041: Link to our GitHub from the Software page.

hjellinek opened the pull request:
I did, and I fixed some typos on the Software and Install and Run page too.

hjellinek committed:
Issue #2041: Link to our GitHub from the Software page.

I did, and I fixed some typos on the Software and Install
and Run page too.

masinter commented on content/en/software/install-and-run/_index.md:
```diff
@@ -11,9 +11,6 @@ weight: 10
-
+Packaged releases of Medley are available for Linux (most recent distros), macOS and Windows 10/11.  Both x86_64 and ARM64 systems are supported (as well as ARM7 - e.g., Raspberry Pi - for Linux).  Instructions for installing and running Medley on these platforms are linked below.
 
+Medley is capable of running on a variety of other OSes including FreeBSD and Solaris, as well as on other CPU architectures including i386, SPARC, PowerPC, and RISC-V.  To install and run Medley on these platforms, you will need to build Medley (and its underlying virtual machine, *Maiko*) from the sources available in the Interlisp repos on [GitHub.com](https://github.com/interlisp/).  Instruct ...
```
you don't have to build medley, just maiko, to port to another system. (Although some features require a posix compatible shell).

You can point at the README more directly by using https://gitnub.com/interlisp/maiko#readme with the `#` fragment identifier.

hjellinek replied:
I updated the text to make it clear that you don't need to build Maiko in order to build Medley.

masinter reviewed (approved):
overlaps with other changes already merged. see comments https://github.com/interlisp/medley/issues/2041

hjellinek committed:
Building Maiko is optional

Clarified that you don't need to build Maiko if you're only building Medley.  (You will need a Maiko instance to run Medley, of course.)

Signed-off-by: Herb Jellinek <jellinek@newscenter.com>

hjellinek committed:
Merge branch 'main' into feature/hj-github-link_issue_2041

Signed-off-by: Herb Jellinek <jellinek@newscenter.com>
